steam/RichPresence.vdf

```diff
@@ -30,13 +30,13 @@
 	{
 		"tokens"
 		{
-			"#StatusTitleScreen"	"W tytulowym menu"
+			"#StatusTitleScreen"	"W menu tytułowym"
 			"#StatusPlayingLevel"	"W poziomie %levelname%"
-			"#StatusEndCard"		"Zkonczył(-a) %levelname% w %levelcompletetime% minuty z oceną %levelcompletegrade%"
+			"#StatusEndCard"		"Skonczył(-a) %levelname% w %levelcompletetime% minuty z oceną %levelcompletegrade%"
 			"#StatusTown"			"W Cornucopia Town"
-			"#StatusWorldMap"		"Podrożuje %worldname%"
-			"#StatusPetMaker"		"W kreatorze zwierzaków"
-			"#StatusLevelBuilder"	"W kreatorze poziomów"
+			"#StatusWorldMap"		"Podróżuje %worldname%"
```

Comment:
I think it might sound better if there was a "w" before %worldname%, but I'm not entirely sure about this one. You might as well keep it the way it is now